packages/scripts/scripts/check-licenses.js

```diff
@@ -193,6 +225,11 @@ modules.forEach( ( path ) => {
 		);
 	let licenseType = typeof license === 'object' ? license.type : license;
 
+	// Check if the license we've detected is telling us to look in the license file, instead.
+	if ( licenseType && licenseFiles.find( ( licenseFile ) => licenseType.indexOf( licenseFile ) >= 0 ) ) {
+		licenseType = undefined;
```

Comment:
What if the `license` property is pointing a file describing a truly incompatible license? Wouldn't this wrongly allow the license to be considered valid? I expect this to be even more common for non-standard licenses.

Reply:
Oh! I missed the code immediately following this, where we try to infer the license type from the presence of a file.

Reply:
🙂